src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -98,55 +102,42 @@ auto partial_derivatives(
                                   tmpl::size_t<Dim>, DerivativeFrame>>;
 
 namespace Tags {
-namespace Tags_detail {
-template <typename Tag, typename VolumeDim, typename Frame>
-struct deriv_impl<Tag, VolumeDim, Frame,
-                  Requires<tt::is_a_v<Tensor, db::item_type<Tag>>>>
-    : db::PrefixTag, db::SimpleTag {
-  using type = TensorMetafunctions::prepend_spatial_index<
-      db::item_type<Tag>, VolumeDim::value, UpLo::Lo, Frame>;
-  using tag = Tag;
-  static std::string name() noexcept { return "deriv"; }
-};
 
-// Partial derivatives with derivative index in the frame related to the logical
-// coordinates by InverseJacobianTag
-template <typename... VariablesTags, typename... DerivativeTags,
-          typename InverseJacobianTag>
-struct deriv_impl<
-    tmpl::list<VariablesTags...>, tmpl::list<DerivativeTags...>,
-    InverseJacobianTag,
-    Requires<tt::is_a_v<Tensor, db::item_type<InverseJacobianTag>>>>
-    : db::ComputeTag {
+/*!
+ * \ingroup DataBoxTagsGroup
+ * \brief Compute the spatial derivatives of tags in a Variables
+ *
+ * Computes the spatial derivatives of the Tensors in the Variables represented
+ * by `VariablesTag` in the frame mapped to by `InverseJacobianTag`. You can
+ * optionally specify a subset of the tags in the Variables as `GradientTags` to
```

Comment:
How do I specify this subset? The way you've worded the documentation it sounds like I'm supposed to have those tags inherit off of a class call `GradientTags`, not specify a typelist.